proxy/http/HttpTransact.cc

```diff
@@ -891,7 +891,6 @@ HttpTransact::SelfLoop(State *s)
 {
   TxnDebug("http_trans", "Request will selfloop.");
   bootstrap_state_variables_from_request(s, &s->hdr_info.client_request);
-  build_error_response(s, HTTP_STATUS_BAD_REQUEST, "Direct self loop detected", "request#cycle_detected");
```

Comment:
Is this line intentionally removed ?

Reply:
Yes this is not related to the fix. This is a cleanup while I am here and I noted that in the description.